docs/logging.rst

```diff
@@ -128,10 +128,9 @@ provided in ``misc/incident-gatherer/support_classifiers.py`` . There is
 roughly one category for each ``log.WEIRD``-or-higher level event in the
 Tahoe source code.
 
-The incident gatherer is created with the "``flogtool
-create-incident-gatherer WORKDIR``" command, and started with "``tahoe
-start``". The generated "``gatherer.tac``" file should be modified to add
-classifier functions.
+The incident gatherer is created with the "``flogtool create-incident-gatherer
+WORKDIR``" command, and started with "``tahoe run``". The generated
+"``gatherer.tac``" file should be modified to add classifier functions.
```

Comment:
Working examples are great.  I guess this is a pre-existing and unrelated defect in the docs, though, so it shouldn't strictly be a blocker for merging this PR.  I can file a ticket for this and the other error in this doc ... but my preferred outcome would probably be to escape Foolscap entirely and no longer have an "incident gatherer" or a "[log] gatherer" (and why is Tahoe-LAFS documenting the Foolscap `flogtool` anyway instead of referencing Foolscap-maintained docs anyway)?

Also there's the larger question of CI for examples which we should have sometime.

So, gonna leave this broken but I filed a ticket - https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3556